MetaData/data/cross_sections.json

```diff
@@ -262,77 +318,56 @@
-    "DiPhotonJetsBox_M40_80-Sherpa" : { "xs" : 303.2, "itype":10 },
-    "DiPhotonJetsBox_MGG-80toInf_13TeV-Sherpa" : { "xs" : 84.4, "itype":10 },
-    "DiPhotonJets_MGG-80toInf_13TeV_amcatnloFXFX_pythia8" : { "xs" : 135.1 },
-    "DiPhotonJetsBox2BJets_MGG-80toInf_13TeV-Sherpa" : {"xs" : 0.494},
-    "DiPhotonJetsBox1BJet_MGG-80toInf_13TeV-Sherpa" : {"xs" : 0.8674276},
-    "QCD_Pt-30to40_DoubleEMEnriched_MGG-80toInf_TuneCUETP8M1_13TeV_Pythia8" : {"xs" : 22110.0 , "br" : 1.0,"itype":30 },
-    "QCD_Pt-40toInf_DoubleEMEnriched_MGG-80toInf_TuneCUETP8M1_13TeV_Pythia8" : {"xs" : 113400.0 , "br" : 1.0,"itype":32 },
-    "QCD_Pt-30toInf_DoubleEMEnriched_MGG-40to80_TuneCUETP8M1_13TeV_Pythia8" : {"xs" : 260500.0 , "br" : 1.0,"itype":31 },      
-    "QCD_Pt-30to40_DoubleEMEnriched_MGG-80toInf_TuneCP5_13TeV_Pythia8" : {"xs" : 24810.0 , "br" : 1.0,"itype":30 },
-    "QCD_Pt-40toInf_DoubleEMEnriched_MGG-80toInf_TuneCP5_13TeV_Pythia8" : {"xs" : 118100.0 , "br" : 1.0,"itype":32 },
-    "QCD_Pt-30toInf_DoubleEMEnriched_MGG-40to80_TuneCP5_13TeV_Pythia8" : {"xs" : 241400.0 , "br" : 1.0,"itype":31 },
-    "GJets_HT-40To100_TuneCUETP8M1_13TeV-madgraphMLM-pythia8" :  { "xs" : 23100.0, "itype" : 4 },
-    "GJets_HT-100To200_TuneCUETP8M1_13TeV-madgraphMLM-pythia8" : { "xs" : 9110.0,  "itype" : 5 },
-    "GJets_HT-200To400_TuneCUETP8M1_13TeV-madgraphMLM-pythia8" : { "xs" : 2280.0,  "itype" : 6 },
-    "GJets_HT-400To600_TuneCUETP8M1_13TeV-madgraphMLM-pythia8" : { "xs" :273.0 , "itype" : 7 },
-    "GJets_HT-600ToInf_TuneCUETP8M1_13TeV-madgraphMLM-pythia8" : { "xs" :94.5 , "itype" : 8 },
-    "GJets_HT-40To100_TuneCP5_13TeV-madgraphMLM-pythia8" :  { "xs" : 18640.0, "itype" : 4 },
-    "GJets_HT-100To200_TuneCP5_13TeV-madgraphMLM-pythia8" : { "xs" : 8631.0,  "itype" : 5 },
-    "GJets_HT-200To400_TuneCP5_13TeV-madgraphMLM-pythia8" : { "xs" : 2185.0,  "itype" : 6 },
-    "GJets_HT-400To600_TuneCP5_13TeV-madgraphMLM-pythia8" : { "xs" :257.7 , "itype" : 7 },
-    "GJets_HT-600ToInf_TuneCP5_13TeV-madgraphMLM-pythia8" : { "xs" :85.4 , "itype" : 8 },
+
+    "TTJets_TuneCUETP8M2T4_13TeV-amcatnloFXFX-pythia8" : { "xs" : 830 },
+    "TTJets_TuneCUETP8M1_13TeV-madgraphMLM-pythia8" : { "xs" : 665 },
+    "TTJets_DiLept_TuneCUETP8M1_13TeV-madgraphMLM-pythia8" : { "xs" : 56.86 },
+    "TTGJets_TuneCUETP8M1_13TeV-amcatnloFXFX-madspin-pythia8" : { "xs" : 3.819 },
+    "TTGG_0Jets_TuneCUETP8M1_13TeV_amcatnlo_madspin_pythia8" : { "xs" : 0.01731 },
+    "TTGG_0Jets_TuneCP5_13TeV_amcatnlo_madspin_pythia8" : { "xs" : 0.01687 },
+    "TGJets_TuneCUETP8M1_13TeV_amcatnlo_madspin_pythia8" : { "xs" : 2.967 },
+    "TGJets_TuneCP5_13TeV_amcatnlo_madspin_pythia8" : { "xs" : 3.055 },
+    "TTTo2L2Nu_TuneCP5_13TeV-powheg-pythia8" : { "xs" : 687.1  },
+    "TTGJets_TuneCP5_13TeV-amcatnloFXFX-madspin-pythia8" : { "xs" : 4.078 },
+    "TTTo2L2Nu_TuneCUETP8M2_ttHtranche3_13TeV-powheg-pythia8" : {"xs" : 87.31, "br" : 1.0},
```

Comment:
Some Top backgrounds seem to miss with respect to removed ones like single top. Can you check it?